packages/experiments/src/components/Toggle/Toggle.state.ts

```diff
@@ -12,6 +12,15 @@ export class ToggleState extends BaseState<IToggleProps, IToggleViewProps, ITogg
       controlledProps: ['checked', 'text'],
       transformViewProps: (newProps: IToggleViewProps) => {
         newProps.text = newProps.checked ? props.onText : props.offText;
+
+        // If the user provides an ariaLabel, then let that be the label for the Toggle.
+        if (!props.ariaLabel) {
+          const ariaLabelText = typeof props.label === 'string' ? props.label : '';
+          const ariaCheckedText = newProps.checked ? props.onText : props.offText;
+          const ariaSeparator = ariaLabelText && ariaCheckedText ? ': ' : '';
+          newProps.ariaLabel = ariaLabelText + ariaSeparator + (ariaCheckedText ? ariaCheckedText : '');
```

Comment:
Nitpick, instead of `ariaCheckedText ? ariaCheckedText : ''` I'd recommend `ariaCheckedText || ''` (same in a couple other places in this PR)